### PR TITLE
changed find-trust-route to typescript plus added more tests cases

### PR DESCRIPTION
**Pull request by FahadDarw**
**What is the change?**
The changes include converting the find trust route from JavaScript to TypeScript and adding additional tests to the automation suite. The added tests cover the following scenarios.

1. Not entering a trust should result in an error
2. Getting too many search results should result in a warning (sch is good for this)
3. Search for a valid trust should find multiple results
4. selecting the trust should populate it in the search
5. clicking continue should populate the trust page
6. Trust page should have
7. Title of the academy

**Why do we need the change?**
These changes were implemented to enhance the automated testing process and to ensure that all possible scenarios are covered.

Webpack has also been updated as well as the login. 

**What is the impact?**
The impact of these changes is minimal, but they will result in a more comprehensive and robust test suite.

**Azure DevOps Ticket**
[User Story 124032](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_workitems/edit/124032): Automation Improvements - Trust search

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/support/commands.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ import "cypress-axe";
 import caseApi from "../api/caseApi";
 import concernsApi from "../api/concernsApi";
 import { AuthenticationInterceptor } from "../auth/authenticationInterceptor";
+import {AuthenticationComponent} from  "../auth/authenticationComponent";
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/support/commands.js:35]: Need to revert these changes, as it will switch it back to the old authentication method

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]

**Review by mikestock-nimble — APPROVED** [review]

**Review by chrisdexnimble — APPROVED** [review]
